pypeit/core/coadd.py

```diff
@@ -639,7 +639,7 @@ def rej_bad(smask, badchi, n_grow_mask, ispec):
             idx = gd[srt]
             # The following may eliminate bright, narrow emission lines
             good, spl = utils.robust_polyfit_djs(waves[idx], flux[idx], 3, function='bspline',
-                    sigma=sig[gd][srt], lower=cr_bsigma, upper=cr_bsigma, use_mad=False)
+            sigma=sig[gd][srt], lower=cr_bsigma, upper=cr_bsigma, use_mad=False)
```

Comment:
I'd prefer continuations of lines be indented.  I tend to indent to the parenthesis of the function; e.g., 
```
good, spl = utils.robust_polyfit_djs(waves[idx], flux[idx], 3, function='bspline',
                                     sigma=sig[gd][srt], lower=cr_bsigma,
                                     upper=cr_bsigma, use_mad=False)
```

Reply:
Yeah I agree that looks better.